### PR TITLE
feat: laravel 11 support

### LINKED ISSUE #199: Update to Laravel 11
Waiting on [testbench](https://github.com/orchestral/testbench/issues/397).

### PR DESCRIPTION
**Pull request by taka-oyama**
Closes #199

### CODE REVIEW

File: tests/Query/BuilderTest.php [hunk 1]
```diff
@@ -568,7 +568,7 @@ public function testInsertDatetime(): void
         $qb = $conn->table($tableName);
 
         $row = $this->generateTestRow();
-        $carbonMax = Carbon::maxValue();
+        $carbonMax = Carbon::create(9999, 12, 31, 23, 59, 59, 'UTC');
```
Comment (kt81) [inline tests/Query/BuilderTest.php:571]: memo: The maxValue and minValue functions were removed when Carbon v3 was introduced.
https://github.com/briannesbitt/Carbon/commit/a4726ae7633a4ae361a2561d2256900115ef2015

Laravel 11 now allows v3 series.
https://github.com/laravel/framework/blob/11.x/composer.json#L39

**Review by kt81 — APPROVED** [review]

File: src/Schema/Blueprint.php [hunk 2]
```diff
@@ -127,7 +127,7 @@ public function uuid($column = 'uuid')
      * @param int|null $length
      * @return ColumnDefinition
      */
-    public function binary($column, $length = null)
+    public function binary($column, $length = null, $fixed = false)
```
Comment (halnique) [inline src/Schema/Blueprint.php:130]: You should add the type specification of the added arguments to the Doc.

**Review by halnique — CHANGES_REQUESTED** [review]

File: composer.json [hunk 3]
```diff
@@ -11,16 +11,16 @@
     "php": "^8.1",
```
Comment (taka-oyama) [inline composer.json:11]:
```suggestion
    "php": "^8.2",
```

File: src/Query/ArrayValue.php [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+namespace Colopl\Spanner\Query;
+
+/**
+ * Just a temp object wrapper so UPDATE statements can handle array types
+ */
+final readonly class ArrayValue
```
[the remark this answers is not in the document]
Comment (taka-oyama) [reply src/Query/ArrayValue.php:23]: Laravel 11 requires 8.2 anyway so composer's PHP version has been updated to 8.2. Thanks.

File: src/Schema/Blueprint.php (hunk 2 above)
Comment (taka-oyama) [reply src/Schema/Blueprint.php:130]: Fixed. Thanks.

**Review by kt81 — APPROVED** [review]
LGTM again

**Review by halnique — APPROVED** [review]

**Pull request merged**